src/pyodide-js/src/util.ts

```diff
@@ -0,0 +1,113 @@
+import { PyodideModule } from "./types";
+
+// Regexp for validating package name and URI
+const PACKAGE_REGEX = '[a-z0-9_][a-z0-9_\-]*';
+const PACKAGE_URI_REGEXP = new RegExp('^https?://.*?(' + PACKAGE_REGEX + ').js$', 'i');
+const PACKAGE_NAME_REGEXP = new RegExp('^' + PACKAGE_REGEX + '$', 'i');
+
+const PUBLIC_API = [
+  'globals',
+  'loadPackage',
+  'loadedPackages',
+  'pyimport',
+  'repr',
+  'runPython',
+  'runPythonAsync',
+  'version',
+  'autocomplete',
+];
```

Comment:
This doesn't belong in this file. I'd say it's important enough to go in a file by itself, `api.ts` or something.